src/payment/dto/create-payment.dto.ts

```diff
@@ -1 +1,5 @@
-export class CreatePaymentDto {}
+export class CreatePaymentDto {
+  userId: string;
```

Comment:
W schemacie jest User a tutaj jest string

Reply:
Miałeś na myśli userId: Number zamiast userId: string? Czy tak właśnie nie oznaczamy wszystkich id'ków? @DariuszKny

Reply:
Tutaj pobieramy całego usera, więc `user` zamiast `userId` i importujemy `UserInterface` z `user/interfaces` a nie schemat. Schemat usera pobieramy do schematu payment, bo schemat jest powiązany z już z samą bazą danych i określa też czy inne informację (np. required), których w tej klasie nie potrzebujemy,  tutaj mamy tylko zamodelować same pola usera, więc wystarczy interfejs. Mam nadzieję, że w miarę jasno to wytłumaczyłam ;D